index.cjs

```diff
@@ -0,0 +1,101 @@
+import Book from './modules/Book';
+import Books from './modules/Books';
+import showSection from './modules/Sections';
+import { DateTime } from './modules/luxon';
```

Comment:
- [ ] Please fix the import statement path here instead of Book use Book.js
because your app gives me this error
![image](https://user-images.githubusercontent.com/74744787/178125384-5d3d9705-2410-4148-8f53-a4d38018f129.png)

Reply:
Thank you, I forgot that without the webpack you have to add the ".js" at the end